trace/trace_test.go

```diff
@@ -126,6 +126,30 @@ func TestSQLStartFinishSpan(t *testing.T) {
 	}, rawSpan.Tags())
 }
 
+func TestEsStartFinishSpan(t *testing.T) {
```

Comment:
You should follow the naming of the function since you have only one test e.g. `func TestEsSpan`